frontend/src/metabase/visualizations/echarts/cartesian/option/axis.ts

```diff
@@ -337,6 +338,11 @@ export const buildCategoricalDimensionAxis = (
       ...getDimensionTicksDefaultOption(settings, renderingContext),
       ...getHistogramTicksOptions(chartModel, settings, chartMeasurements),
       formatter: (value: string) => {
+        const numberValue = parseNumberValue(value);
+        if (numberValue !== null) {
```

Comment:
I believe we should check the column type here using ` isNumber` from `"metabase-lib/v1/types/utils/isa"`, otherswise ticks for `["Metabase", "1Password"]` will be `Metabase, 1`